### ISSUE TITLE
RSS page crashes

### ISSUE THREAD

**Issue by igor-shevchenko** [adelnizamutdinov/TawkTube#32]
Thank for a great and useful service!

RSS feed page right now returns 500 for every playlist. Example: http://tawktube.com/playlist?list=PLfbxrJr-t5mmbfVGG01Gd8Yh-njCgcOFK

**Comment by adelnizamutdinov** [comment]
yeah, we're hitting youtube api rate limiting. I'll see what I can do. Thanks for reporting!

**Comment by dpk4g** [comment]
I subscribed several youtube channels and listening on pocket cast, but it has been abnormal in recent days for every channel.
The channel's data can be updated but couldn't playing,it's always show "buffering" and can't download audio also.
I can’t figure out where it’s wrong.
Thank for a great service too.

**Comment by adelnizamutdinov** [comment]
@dpk4g hey! Thanks for reporting. What OS and podcast player do you use to play the feeds?

**Comment by dpk4g** [comment]
I am using Pocket casts and AntennaPod to play the feeds on Android 8.1 OS，none of them can work properly.
![Screenshot_2019_08_06_01_11_02_634_au](https://user-images.githubusercontent.com/52632965/62482403-a11ef600-b7e7-11e9-972c-adb2fae11d34.jpg)
![IMG_20190806_011038](https://user-images.githubusercontent.com/52632965/62482412-a5e3aa00-b7e7-11e9-9cea-4d1ee7501330.jpg)

My subscriptions example：http://tawktube.com/channel/UCwb7avxK-L5vPjMC1ZIGayw , http://tawktube.com/channel/UCa6ERCDt3GzkvLye32ar89w.
btw, I used VPN to connect Internet, so i think it shouldn't  be a problem with the network block.
Thanks for your response.

**Comment by fundamentalista** [comment]
since a few days, i am getting error 503, using android 8.1 and 5.0.1, both with Podcast Republic and Padcast Addict

**Comment by adelnizamutdinov** [comment]
yeah, YouTube changed some of it's apis. I'll deal with it in #33. This issue is about a different problem
